src/gui/utils/colorutils.h

```diff
@@ -0,0 +1,164 @@
+/* This is the stripped version of the file kcolorutils.h
```

Comment:
@sledgehammer999  was against dependencies on KDE/KF5, that is why in this PR we read config files manually and interpret them